apps/files_sharing/src/views/SharingInherited.vue

```diff
@@ -128,8 +128,8 @@ export default {
 		async fetchInheritedShares() {
 			this.loading = true
 			try {
-				const url = generateOcsUrl(`apps/files_sharing/api/v1/shares/inherited?format=json&path=${encodeURIComponent(this.fullPath)}`, 2)
-				const shares = await axios.get(url.replace(/\/$/, ''))
+				const url = generateOcsUrl(`apps/files_sharing/api/v1/shares/inherited?format=json&path=${encodeURIComponent(this.fullPath)}`)
```

Comment:
Ah there are a few other cases like this that I can adjust:
```php
const url = generateOcsUrl(`apps/files_sharing/api/v1/shares/inherited?format=json&path=${encodeURIComponent(this.fullPath)}`)
```
=>
```php
const url = generateOcsUrl('apps/files_sharing/api/v1/shares/inherited?format=json&path={path}', { path: encodeURIComponent(this.fullPath) })
```
Just out of interest, since the first/current looks simpler, what is the benefit of the second?

Reply:
The following does not work?
```suggestion
				const url = generateOcsUrl('apps/files_sharing/api/v1/shares/inherited', { format: 'json', path: encodeURIComponent(this.fullPath) })
```

If not, just extract it:
```suggestion
				const url = generateOcsUrl('apps/files_sharing/api/v1/shares/inherited') + '?format=json&path=' + encodeURIComponent(this.fullPath)
```

Reply:
Nope, it does not append parameters as query string value: key pairs like that: https://github.com/nextcloud/nextcloud-router/blob/8bce573e13375fc425486af12807b7d93b60b468/lib/index.ts#L39-L89
Would be smart, actually, although probably not good to do too much magic in such library functions so that usage needs to be more explicit but outcome better predictable 😄.

Okay so you prefer concatenating strings over this backticks execution, I guess as best practice security-wise as well?

Reply:
Ah why not including the first part of the query string into the function?
```php
generateOcsUrl('apps/files_sharing/api/v1/shares/inherited?format=json&path=') + encodeURIComponent(this.fullPath)
```